examples/android/README.md

```diff
@@ -1,12 +1,14 @@
 # Flower Android Example (TensorFlowLite)
 
-This example demonstrates a federated learning setup with Android Clients. The training on Android is done on a CIFAR10 dataset using TensorFlow Lite. The setup is as follows:
+This example demonstrates a federated learning setup with Android Clients in the background thread. The training on Android is done on a CIFAR10 dataset using TensorFlow Lite. The setup is as follows:
 
 - The CIFAR10 dataset is randomly split across 10 clients. Each Android client holds a local dataset of 5000 training examples and 1000 test examples.
 - The FL server runs in Python but all the clients run on Android.
 - We use a strategy called FedAvgAndroid for this example.
 - The strategy is vanilla FedAvg with a custom serialization and deserialization to handle the Bytebuffers sent from Android clients to Python server.
 
+This background thread is established via WorkManager library of Android, thus it will run comfortably on Android Versions from 8 to 13.
```

Comment:
```suggestion
The background thread is established via the `WorkManager` library of Android, thus, it will run comfortably on Android Versions from 8 to 13.
```